app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue

```diff
@@ -291,6 +291,24 @@ export default {
   overflow: hidden;
 }
 
+.app-rtl--wrapper {
+  .contacts-table-wrap {
+    &::v-deep {
+      .ve-table-content {
+        .ve-table-body .ve-table-body-tr .ve-table-body-td {
+          text-align: right !important;
```

Comment:
Can we avoid the important?

Reply:
@nithindavid I tried it but not working

Reply:
Done https://github.com/chatwoot/chatwoot/pull/6525/commits/b1e55901bcc1b5ce8d5116a21fdbd9604470e741